src/ShowGuesser.test.js

```diff
@@ -6,6 +6,8 @@ import FieldGuesser from './FieldGuesser';
 import { API_FIELDS_DATA } from './__fixtures__/parsedData';
 import { IntrospectedShowGuesser } from './ShowGuesser';
 
+jest.mock('./useMercureSubscription');
```

Comment:
Inclusion of `useMercureSubscription` bring in a dependency on redux. Problem is, react-redux v6+ [doesn't play nicely with enzyme](https://github.com/enzymejs/enzyme/issues/2282)

What I've tried:
```js
const DummyReduxProvider = ({ children }) => (
  <Provider store={createStore(() => {})}>{children}</Provider>
);

const wrapper = shallow(
  <IntrospectedShowGuesser
    resource="user"
    schema={{ name: 'users', title: 'User' }}
    readableFields={API_FIELDS_DATA}
    id="ShowComponentId"
  />,
  {
    wrappingComponent: DummyReduxProvider,
  },
);
```